scss/navbar.scss

```diff
@@ -0,0 +1,25 @@
+.navbar {
```

Comment:
this should have the 20px padding you added in the logo and in the right title, since it's the navbar separating its limits from its children (the logo and the text)